src/Elmish.WPF/Elmish.WPF.fsproj

```
@@ -15,12 +27,24 @@
     <Compile Include="Utils.fs" />
     <Compile Include="Cmd.fs" />
     <Compile Include="Program.fs" />
-    <None Include="paket.references" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Elmish" Version="3.0.3" />
```

Would be good to keep the version constraint from `paket.dependencies` to avoid unintentional updates to a potentionally breaking Elmish v4.

---

Good idea. Do you know the exact syntax?

---

This should work:

```suggestion
    <PackageReference Include="Elmish" Version="[3.0.3, 4)" />
```

<https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges-and-wildcards>

---

Perfect, thanks!